resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/KubernetesConfSuite.scala

```diff
@@ -218,6 +218,14 @@ class KubernetesConfSuite extends SparkFunSuite {
     assert(driverConf.schedulerName === "driverScheduler")
   }
 
+  test("SPARK-36059: access appId in KubernetesConf") {
```

Comment:
Given that the PR title is `SPARK-37735`, this `SPARK-36059` looks wrong to me.

Reply:
@dongjoon-hyun Thanks, it was bad copy paste. Fixed.